python/tests/evaluation/test_evaluation.py

```diff
@@ -41,7 +41,6 @@ def predict(inputs: dict) -> dict:
         },
         num_repetitions=3,
     )
-    results.wait()
```

Comment:
why was this removed?

Reply:
Wasn't necessary since we were blocking in the function anyway